items/items_init.py

```diff
@@ -0,0 +1,28 @@
+from items.weapon import Weapon
+from items.armour import Armour
+from items.potion import Potion
+
+
+class InitItems:
+
+    @staticmethod
+    def init_weapon(name: str, config: dict) -> Weapon:
+        weapon_name = config.get(name).get("weapon").get("name")
+        power = config.get(name).get("weapon").get("power")
+        return Weapon(weapon_name, power)
+
+    @staticmethod
+    def init_armour(name: str, config: dict) -> list[Armour]:

```

Comment:
Its not really initialization, but creation, because you create Armour object here